php-classes/Slate/CBL/DemonstrationSkill.php

```diff
@@ -22,6 +22,9 @@ class DemonstrationSkill extends \ActiveRecord
             'type' => 'tinyint',
             'unsigned' => true
         ]
+        ,'Demonstrated' => [
+            'type' => 'date'
+        ]
```

Comment:
@jmealo this field shouldn't be getting added to the DemonstrationSkill model, it's already stored in the related Demonstration model linked by DemonstrationID